tests/shiftenter.js

```diff
@@ -26,6 +27,12 @@ describe( 'ShiftEnter feature', () => {
 		expect( editor.commands.get( 'shiftEnter' ) ).to.be.instanceof( ShiftEnterCommand );
 	} );
 
+	it( 'registers the EnterObserver', () => {
```

Comment:
To make this test secure and show what it really tests, we should also check that the `Enter` plugin is not available. Then, it'll be completely clear that `ShiftEnter` should do it on purpose.